app/views/ish_manager/videos/_index.haml

```diff
@@ -1,31 +1,15 @@
 
 - videos ||= @videos
-- videos = videos.page params[:videos_page]
+- videos = videos.page(params[:videos_page]).per( 10 )
```

Comment:
let's use this variable instead of 10: https://github.com/piousbox/ish_models/blob/master/lib/video.rb#L6